go/enclave/db/interfaces.go

```diff
@@ -16,8 +16,10 @@ import (
 type BlockResolver interface {
 	// FetchBlock returns the L1 Block with the given hash.
 	FetchBlock(blockHash common.L1RootHash) (*types.Block, error)
-	// StoreBlock persists the L1 Block
-	StoreBlock(block *types.Block)
+	// StoreL1HeadBlock stores L1 blocks (which should only be stored when they are the head of the chain that is known to the enclave)
```

Comment:
Can you make it clear here that it stores the L1 block *and* updates the L1 head?